pkg/osquery/tables/dataflattentable/exec.go

```diff
@@ -53,7 +55,23 @@ func TablePluginExec(client *osquery.ExtensionManagerClient, logger log.Logger,
 		panic("Unknown data source type")
 	}
 
-	return table.NewPlugin(t.tableName, columns, t.generateExec)
+	generateFunc := t.generateExec
+
+	// if the binary doesn't exists, no-op the table's generate function.
+	if _, err := os.Stat(t.execArgs[0]); err != nil {
+		level.Info(t.logger).Log("msg", "unable to find binary to exec", "err", err)
```

Comment:
I think there's a cleaner place for this... The exec can fail for any number of reasons. We should populate the `results` with the error after `t.exec(ctx)`

Reply:
I'm not sure we even need the stat -- `exec.CommandContext` should handle it. Just push the error along.